code/modules/modular_computers/computers/modular_computer/assembly_computer.dm

```diff
@@ -0,0 +1,92 @@
+/*
+ *
+ * Assembly explicitly for modular computers.
+ *
+ */
+/datum/extension/assembly/modular_computer
+	var/max_hardware_size
+	var/hardware_flag
+	var/bsod
+	assembly_name = "computer"
+	max_parts = list(
+		PART_BATTERY 	= 1,
+		PART_CPU		= 1,
+		PART_NETWORK	= 1,
+		PART_HDD		= 1,
```

Comment:
Have you tried actually using the removable storage? I have a suspicion it might not be working because PART_DRIVE is actually subtype of PART_HDD so it might be getting blocked by the max parts check.

Reply:
You're right. It is blocked.

Reply:
I have fixed this now.